package.json

```diff
@@ -14,7 +14,7 @@
 		"atom": ">=1.34.0"
 	},
 	"scripts": {
-		"lint": "xo --ignore='mocks/**/*'",
+		"lint": "tsc --build && xo",
```

Comment:
The `tsc --build` is doing the type checking, as babel doesn't do any. So we still need `typescript` as a dependency